packages/core/lib/command-utils.js

```diff
@@ -0,0 +1,209 @@
+const TaskError = require("./errors/taskerror");
+const { bundled, core } = require("../lib/version").info();
+const OS = require("os");
+const analytics = require("../lib/services/analytics");
+const { extractFlags } = require("./utils/utils"); // contains utility methods
+const globalCommandOptions = require("./global-command-options");
+const debugModule = require("debug");
+const debug = debugModule("core:command:run");
+const commands = require("./commands/commands");
+
+const getCommand = (inputStrings, options, noAliases) => {
+  if (inputStrings.length === 0) {
```

Comment:
Is this `inputStrings` here the same as the one in `cli.js`? if so, when the `inputString` length is 0, wouldn't it qualify as `userWantsGeneralHelp` instead of throwing the error?

Reply:
Yes, that's true! I guess we also need to consider this utility function on its own as you never know where it might be used later on. It is probably ok for this to throw when no input is given, eh? 

That makes me take note though, I think we need to put the same kind of check for 0 input strings in the console context. I don't think that exists right now.